src/components/App.js

```diff
@@ -1733,6 +1733,14 @@ class App extends Component {
         this.setCurrentTreePosition(newTree, treePosition, {clearCache: true})
     }
 
+    swapColors() {
```

Comment:
Regarding nomenclature, I think we should stick with "invert colors" to have it the same as digital painting/photo editing software.